android/src/main/java/com/opentokreactnative/OTSubscriberLayout.java

```diff
@@ -26,11 +26,13 @@ public OTSubscriberLayout(ThemedReactContext reactContext) {
     public void createSubscriberView(String streamId) {
 
         ConcurrentHashMap<String, Subscriber> mSubscribers = sharedState.getSubscribers();
-        String pubOrSub = sharedState.getAndroidOnTop();
-        String zOrder = sharedState.getAndroidZOrder();
+        ConcurrentHashMap<String, String> androidOnTopMap = sharedState.getAndroidOnTopMap();
+        ConcurrentHashMap<String, String> androidZOrderMap = sharedState.getAndroidZOrderMap();
         Subscriber mSubscriber = mSubscribers.get(streamId);
         FrameLayout mSubscriberViewContainer = new FrameLayout(getContext());
         if (mSubscriber != null) {
+            String pubOrSub = androidOnTopMap.get(mSubscriber.getSession().getSessionId());
```

Comment:
I would check if this value is null or not

Reply:
Done!